access/src/main/external/static/templates/admin/changeTypeForm.html

```diff
@@ -0,0 +1,22 @@
+<form id="change_type_form" method="post" action="services/api/edit/changeType" class="edit_form change_type_form">
+	<p>
+		<div class="form_field">
+			<label>New Type</label>
+			<select id="change_type_new_type">
+				<option value="Aggregate" <%= "Aggregate Work" == defaultType? "disabled" : "" %>>Aggregate Work</option>
+				<option value="Folder" <%= "Folder" == defaultType? "disabled" : "" %>>Folder</option>
+				<option value="Collection" <%= "Collection" == defaultType? "disabled" : " "%>>Collection</option>
+			</select>
+		</div>
+	
+		<div class="update_field">
+			<input type="submit" id="change_type_submit_button" class="update_button" value="Change" />
```

Comment:
We should standardize on a title for buttons like this. The access controls modal has "Update", this has "Change", and edit label and edit description have "Save Changes".